### ISSUE TITLE
[profile]

### ISSUE THREAD

**Issue by gnutux95** [sosreport/sos#1382]
Hello,
Nice tool, good job, I think that I don't need to reinvent the wheel thanks to this tool....

But i use sosreport currently, i am very surprise with the profile, more exactly the creation. It seems to be not implement for all....

Except my mistake, I discover that we can't create any profile outside the python code without editing as the plugin but it is necessary to edit each necessary plugin.py to add several plugin in one new profile....

The need is there : 
I want to use specific plugin with rewrite the same line in other plugin, but we don't include it, we must use profile. You can add profile in your plugin file, but I need too other standard existing plugin, I can't edit it without do a lot of modification.... what...t..f...
Profile is needed to avoid to rewrite the same thing. 

It is unusual and weird to do that... Why we don't use /etc/sos.conf to configure profile ? Do i need all recode into a plugin (every line coming from other plugin, when I could include other plugin in a profile isn't shall ?
Except if I forget something, we can't do it in a configuration file or inside one file only or in commande ligne which configurer automatically the plugin or others things user side (no developper side)... ?

Have you the good way ? I didn't information on creation profile, only for use with the command :
sosreport -p profile
sosreport --list-profile

not
sosreport --create-profile toto --plugins titi, tutu, tata

or
/etc/sos.conf

line :
[profile]
newprofile = titi,tutu,tata

Regards,
Tux

**Comment by bmr-cymru** [comment]
Use [presets](https://github.com/sosreport/sos/pull/1323) for this:

```
# sosreport --add-preset mypreset -o$MY_PLUGIN_LIST -k$MY_PLUGIN_OPTION_LIST

sosreport (version 3.6)

Added preset 'mypreset' with options -o$MY_PLUGIN_LIST -k$MY_PLUGIN_OPTION_LIST

# sosreport --preset mypreset
[...]
```

You don't mention the version you are running but the preset feature is available in master since `sos-3.6`. Profiles were never intended to be user-editable - they are defined by the plugin authors based on knowledge of related information that may be used together. Now that we have presets it is possible that we may also introduce a "user profile" feature in future that would allow users to name sets of plugins to be run together, however, since it's essentially just a subset of the general preset feature it is not something we're working on at high priority.

**Comment by bmr-cymru** [comment]
You'll need to wait until the next RHEL7 update to have `3.6`, or you can build your own packages in the meantime.
